stackslib/src/chainstate/stacks/db/mod.rs

```diff
@@ -299,14 +299,14 @@ impl DBConfig {
         });
         match epoch_id {
             StacksEpochId::Epoch10 => true,
-            StacksEpochId::Epoch20 => version_u32 >= 1 && version_u32 <= 7,
-            StacksEpochId::Epoch2_05 => version_u32 >= 2 && version_u32 <= 7,
-            StacksEpochId::Epoch21 => version_u32 >= 3 && version_u32 <= 7,
-            StacksEpochId::Epoch22 => version_u32 >= 3 && version_u32 <= 7,
-            StacksEpochId::Epoch23 => version_u32 >= 3 && version_u32 <= 7,
-            StacksEpochId::Epoch24 => version_u32 >= 3 && version_u32 <= 7,
-            StacksEpochId::Epoch25 => version_u32 >= 3 && version_u32 <= 7,
-            StacksEpochId::Epoch30 => version_u32 >= 3 && version_u32 <= 7,
+            StacksEpochId::Epoch20 => version_u32 >= 1 && version_u32 <= 8,
```

Comment:
This is a nit. Should this be set to instead of hard coded 8, "CHAINSTATE_VERSION" ?

Reply:
The idea with the code structured as it currently is, is that when the chainstate schema version changes, they need to make sure that it actually supports the old epochs. I am not a huge fan of this particular switch statement (I don't think its very good at describing the actual compatibility), but I'm loathe to upset the apple cart too much here while I'm making other changes. I'd prefer to do that in a separate PR.